app/services/request_key_manager.rb

```diff
@@ -5,4 +5,11 @@ class RequestKeyManager
       Figaro.env.saml_passphrase
     )
   end
+
+  cattr_accessor :equifax_ssh_key do
```

Comment:
Why not use a constant instead? It's 2.58x times faster according to the benchmark I just ran?